library/src/main/java/net/grandcentrix/tray/core/AbstractTrayPreference.java

```diff
@@ -176,6 +185,7 @@ public void unregisterOnTrayPreferenceChangeListener(
      */
     protected void annex(final T oldStorage) {
         getStorage().annex(oldStorage);
+        logv("annexed " + oldStorage + " to " + this);
```

Comment:
Instead of importing the method I would write ```TrayLog.logv(…``` makes it more obvious that the method isn't part of ```AbstractTrayPreference``` class